FAQ and Troubleshooting article (#5890)
wix/wix-style-react: docs/FAQ/README.md

`import`ing a component directly from `dist/src` is strongly prohibited.

### I'm not sure which component to use to implement the desired spec
We recommend checking out our [cheatsheet](https://www.wix-style-react.com/?path=/story/introduction-cheatsheet--components-cheatsheet) page which includes a view of most of the library components inventory.
Furthermore, it includes links to the components and ux stories. If you cannot find it there and you're unable to find it by the regular search, please contact us at #wix-style-ux or #wix-style-react channels.

### The component I'm using does not look or behave as expected

> Firstly, make sure that you use the component as described within the examples and API description. Should the issue persists,

Reviewer: ```suggestion
Firstly, make sure that you use the component as described within the examples and API description. Should the issue persist,
```

> please try to recreate it within storybook. Once you are able to recreate it, please contact #wix-style-react channel and provide us with the following information:

Reviewer 2: Maybe add a link to the playground here

> please try to recreate it within storybook. Once you are able to recreate it, please contact #wix-style-react channel and provide us with the following information:

Reviewer: ```suggestion
please try to recreate it in the [playground](https://www.wix-style-react.com/?path=/story/introduction-playground--playground). Once you are able to recreate it, please contact #wix-style-react channel and provide us with the following information:
```
